packages/firestore/src/model/field_value.ts

```diff
@@ -235,6 +254,22 @@ export class DoubleValue extends NumberValue {
     }
   }
 
+  toProto(useProto3Json: boolean): api.Value {
+    if (useProto3Json) {
+      // Proto 3 let's us encode NaN and Infinity as string values as
```

Comment:
Drive-by comment: "let's" should be "lets"

Reply:
This was copied, so I got to apply this fixed twice. Thanks.